### ISSUE TITLE
grasp_train.py integrate pixel-wise loss option

### ISSUE THREAD

**Issue by ahundt** [jhu-lcsr/costar_plan#366]
We need to integrate pixel-wise loss into grasp_train.py, implementations are available in keras-contrib but we also need to account for the coordinate of the gripper.

 - [x] pixel wise loss at gripper coordinate
 - [x] gaussian loss at gripper coordinate
 - [x] ensure standard loss functions work too
- [x] add configuration of single pixel to grasp_train
- [ ] add configuration of gaussian to grasp_train
- [ ] add unit test for both
